### DISCUSSION TITLE
Compatibility with latest "maplibre-gl-native" releases?

### DISCUSSION THREAD

**Discussion by christianlask** [rnmapbox/maps#1511] [Q&A]
Hey all! First of all, thanks for providing the "react-native-mapbox-gl/maps" module to the community.. very much appreciated!

We want to use it in a React Native-based application to provide our users with a map view of their location. We have our own OSM tile servers setup and want to use them for obtaining tiles and cannot rely on a third-party service for that.

That's why we wanted to use this module with "MapLibre" in the background and especially with the most recent releases where any dependency on MapBox service has been removed (see [this PR](https://github.com/maplibre/maplibre-gl-native/pull/90)) which has been part of the MapLibre-gl-native releases [9.5.0 for Android](https://github.com/maplibre/maplibre-gl-native/releases/tag/android-v9.5.0) and[ 5.12.0 for iOS](https://github.com/maplibre/maplibre-gl-native/releases/tag/ios-v5.12.0) respectively.

This change leads to an incompatibility with the "react-native-mapbox-gl/maps" library, obviously. Are you aware of this yet? And if so, are there any plans to keep the library compatible with the MapLibre SDKs?

Thanks in advance!

**Comment by ferdicus** [comment]
hi @christianlask, 
yes we're aware since [this issue ticket ](https://github.com/react-native-mapbox-gl/maps/issues/1472)popped up.

Unfortunately we didn't have the time to look into it (not sure how much change from our side is required to get it going again).  
Ignoring the effort that would need to go into supporting MapLibre 5.12.0, it would probably also ensure a departure from MapBox support (correct me here please @mfazekas if you disagree).

**Reply by christianlask** [reply]
Hi @ferdicus!

> yes we're aware since [this issue ticket ](https://github.com/react-native-mapbox-gl/maps/issues/1472)popped up.

And here I was thinking I had done my research ;-)

> Unfortunately we didn't have the time to look into it (not sure how much change from our side is required to get it going again).

  I see, thanks! Since this is a rather pressing issue for us (me, my colleagues and the company I work for) I will ask on [Gitter](https://gitter.im/react-native-mapbox-gl/Lobby) if anyone is willing to restore compatibility while getting paid for doing so :)
